src/redis-brain.js

```diff
@@ -17,14 +20,20 @@ const Url = require('url')
 const Redis = require('redis')
 
 module.exports = function (robot) {
-  let client, prefix
+  let client, prefix, redisUrl
   const redisUrlEnv = getRedisEnv()
-  const redisUrl = process.env[redisUrlEnv] || 'redis://localhost:6379'
 
   if (redisUrlEnv) {
```

Comment:
Should we not have `if (process.env[redisUrlEnv]) {` to remain consistent with previous code?

Reply:
I did attempt to wrap that statement in an `if-else` statement, but then redisUrlEnv becomes only usable in the if-else block and breaks the rest of the code. That's why I went with moving that variable to the `let` statement. Suggestions?